app/callback/sqs_client.py

```diff
@@ -23,7 +23,11 @@ def send_message(self, url: str, message_body: dict, message_attributes: dict =
         message_attributes["ContentType"] = {"StringValue": "application/json", "DataType": "String"}
         try:
             response = self._client.send_message(
-                QueueUrl=url, MessageBody=json.dumps(message_body), MessageAttributes=message_attributes
+                QueueUrl=url,
+                MessageBody=json.dumps(message_body),
+                MessageAttributes=message_attributes,
+                # if SQS is fifo then
+                MessageGroupId=url if 'fifo' in url else None
```

Comment:
You have to be careful with this. We don't know how they interpret "None", that's why I sent you the code I did. If you check their documentation:

>You can't use it for Standard queues.

Best to not include it. So your options are what I sent, an if/else block with duplicate code, or something else.

Reply:
I tried doing similar before with a different module but the default behavior was to set the unassigned parameters as instantiations of a class, so setting it to `None` broke it.